examples/heartbeat-print.cpp

```diff
@@ -17,11 +17,11 @@
  */
 
 #include <arpa/inet.h>
-#include <assert.h>
-#include <errno.h>
-#include <signal.h>
-#include <stdio.h>
-#include <stdlib.h>
+#include <cassert>
```

Comment:
then we mix headers with and without .h ?  never understood the benefit of this "modernization"

Reply:
I have researched the differences again and found no real difference, than "it the C++ style". Will change it back later today or tomorrow.